sentinelhub/config.py

```diff
@@ -144,44 +149,56 @@ def __eq__(self, other: object) -> bool:
         return all(getattr(self, param) == getattr(other, param) for param in self.get_params())
 
     @classmethod
-    def load(cls, filename: Optional[str] = None) -> SHConfig:
+    def load(cls, filename: Optional[str] = None, profile: str = DEFAULT_PROFILE) -> SHConfig:
```

Comment:
filenames now don't make sense anymore, right? do you plan this for a separate MR?

Reply:
in some cases one might want to have a file in a different location i guess?
for instance in the SH-QGIS plugin etc. But yeah, it does complicate things a bit in the loading method.

Reply:
I guess it just feels weird to not have the standard `object.save(file)` and `load_object(file)` interface :thinking:  will sleep it through

Reply:
i'll leave it in for now, and open an issue so that we can reconsider this decision